authui/src/authflowv2/components/input.css

```diff
@@ -0,0 +1,87 @@
+@layer components {
+  :root {
+    --input__ring-color: var(--color-grey-100);
+    --input__ring-color--focus: var(--color-primary-base);
+    --input__ring-color--error: var(--color-error);
+    --input__ring-width: var(--border-radius-regular);
+    --input__ring-width--focus: var(--border-radius-medium);
+    --input__ring-width--error: var(--border-radius-medium);
+    --input__border-radius: var(--border-radius-rounded-square);
+    --input__placeholder-color: var(--color-grey-200);
+    --input__text-color: var(--color-grey-700);
+    --input__font-family: var(--typography-body-large__font-family);
+    --input__font-size: var(--typography-body-large__font-size);
+    --input__line-height: var(--typography-body-large__line-height);
+    --input__letter-spacing: var(--typography-body-large__letter-spacing);
+    --input__font-weight: var(--typography-body-large__font-weight);
+    --input__y-padding: 1em;
+    --input__x-padding: 1.09em;
+
+    --input-error-message__text-color: var(--color-error);
```

Comment:
It should be `--input__error-message-text-color`. `input` is the BEM block. `error-message-text-color` is the BEM element.